Vic2ToHoI4/Source/V2World/States/State.cpp

```diff
@@ -53,6 +53,26 @@ int Vic2::State::determineEmployedWorkersScore(const workerStruct& workers)
 }
 
 
+void Vic2::State::eatState(const State& state, const StateDefinitions& stateDefinitions)
+{
+	provinceNumbers.insert(state.provinceNumbers.begin(), state.provinceNumbers.end());
+	provinces.insert(state.provinces.begin(), state.provinces.end());
+
+	partialState = false;
+	for (const auto provinceNumber: stateDefinitions.getAllProvinces(*provinceNumbers.begin()))
+	{
+		if (!provinceNumbers.contains(provinceNumber))
+		{
+			partialState = true;
+			break;
+		}
+	}
+
+	factoryLevel += state.factoryLevel;
```

Comment:
This could escalate quickly. Imagine a german state being owned by 4 minors who built factories. Then again, maybe it's not an issue as they couldn't likely field people to ramp them up.

Reply:
Yeah, factory level is just a soft cap, so it should come out okay.